src/legacy/core_plugins/kibana/server/tutorials/apm/envs/on_prem.js

```diff
@@ -158,11 +158,10 @@ export function onPremInstructions(apmIndexPattern) {
             index: apmIndexPattern,
             query: {
               bool: {
-                filter: {
-                  exists: {
-                    field: 'processor.name',
-                  },
-                },
+                should: [
+                  { term: { 'processor.name': 'error' } },
+                  { term: { 'processor.name': 'transaction' } },
```

Comment:
`metric` and `sourcemap` processors could also be good enough for green.  sourcemap is unlikely to to be unaccompanied but I expect eventually users may have metrics-only APM

Reply:
Good point. I'll just add both - can't hurt.
I initially thought about making a query that check for documents where `processor.name exists` and `processor.name !== "onboarding"` but I preferred the explicit query. Might make sense to revert this later if we get too many doc types.

Reply:
Yeah, that's what I suggested in the issue report but I'm not fussed either way, thanks for getting this in.

Reply:
Sure! From Discuss it's clear that it causes a lot of confusion to users so I bumped the priority.